sql/sql_show.cc

```diff
@@ -5502,11 +5500,11 @@ static ST_FIELD_INFO user_stats_fields_info[] = {
      MY_I_S_UNSIGNED, "Total_connections", SKIP_OPEN_TABLE},
     {"CONCURRENT_CONNECTIONS", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_LONGLONG,
      0, MY_I_S_UNSIGNED, "Concurrent_connections", SKIP_OPEN_TABLE},
-    {"CONNECTED_TIME", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_LONGLONG, 0,
+    {"CONNECTED_TIME", MY_INT64_NUM_DECIMAL_DIGITS, MYSQL_TYPE_DOUBLE, 0,
```

Comment:
There are testcases that check I_S schema - please grep the testsuite for the changed columns

Reply:
MY_INT64_NUM_DECIMAL_DIGITS might work as display length for double too but even then it needs a better mnemonic

Reply:
I kept `MY_INT64_NUM_DECIMAL_DIGITS` because MariaDB did so.